dojo/tools/dockle/parser.py

```diff
@@ -37,7 +37,9 @@ def get_findings(self, filename, test):
                 severity = self.SEVERITY[dockle_severity]
             else:
                 severity = "Medium"
-            description = "\n".join(item.get('alerts', []))
+            description = item.get('alerts', [])
+            description.sort()
+            description = "\n".join(description)
```

Comment:
```suggestion
            description = "\n".join(item.get('alerts', []).sort())
```

Reply:
Good suggestion here

Reply:
I've had already tried this way but it doesn't work because the .sort() function returns `NoneType`.
You must have 3 separate ugly lines but at least it works.
![image](https://user-images.githubusercontent.com/47610842/127618420-297e53ed-6d56-4019-ac18-318b6c219c3e.png)

Reply:
I've had already tried this way but it doesn't work because the .sort() function returns NoneType.
You must have 3 separate ugly lines but at least it works.
![Capture d’écran 2021-07-30 à 09 41 34](https://user-images.githubusercontent.com/47610842/127618892-d74d7631-090b-435e-8be8-b90b11316adb.png)
![Capture d’écran 2021-07-30 à 09 36 37](https://user-images.githubusercontent.com/47610842/127618893-1eee5cb7-0333-4599-adab-9458b56dfd95.png)